server/sonar-db-dao/src/test/java/org/sonar/db/rule/RuleDbTester.java

```diff
@@ -111,12 +118,43 @@ public RuleDto insertRule() {
     });
   }
 
-  public RuleDto insertRule(OrganizationDto organization, Consumer<RuleDto>... populaters) {
+  @SafeVarargs
+  public final RuleDto insertRule(OrganizationDto organization, Consumer<RuleDto>... populaters) {
     RuleDto ruleDto = newRuleDto(organization);
     Arrays.asList(populaters).forEach(populater -> populater.accept(ruleDto));
     return insertRule(ruleDto);
   }
 
+  @SafeVarargs
+  public final RuleDefinitionDto insertRuleDefinition(Consumer<RuleDefinitionDto>... populaters) {
```

Comment:
This method already exists, see insert(Consumer). I think we don't need varargs. A single consumer is enough.